apps/src/aichat/types.ts

```diff
@@ -16,8 +16,7 @@ export type ChatCompletionMessage = {
 };
 
 export type AichatContext = {
-  userId: number;
-  currentLevelId: string | null;
+  currentLevelId: number | null;
```

Comment:
Removed sending user ID from the front end because we should be able to derive the user id from the signed in user, and don't want to allow one user to update another user's history.